src/test/operate.test.js

```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
```

Comment:
Kindly write tests for all operations with the `calculate.js` component as well.

For example:

```
import calculate from '../../logic/calculate';

describe('testing the calculate function ', () => {
  const obj = { total: null, next: null, operation: null };

  test('test addition', () => {
    let result = calculate(obj, '4');
    result = calculate(result, '+');
    result = calculate(result, '7');
    result = calculate(result, '=');

    expect(result.total).toBe('11');
  });

  test('test subtraction', () => {
    let result = calculate(obj, '8');
    result = calculate(result, '-');
    result = calculate(result, '3');
    result = calculate(result, '=');

    expect(result.total).toBe('5');
  });

... and so forth
end
```